dev/io.openliberty.microprofile.telemetry.2.0.logging.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal/container/fat/JULLogServletTest.java

```diff
@@ -0,0 +1,102 @@
+import componenttest.containers.SimpleLogConsumer;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+
+@RunWith(FATRunner.class)
+public class JULLogServletTest {
+
+    private static Class<?> c = JULLogServletTest.class;
+
+    @Server("TelemetryAgentJULMessages")
+    public static LibertyServer server;
+
+    private static final String[] EXPECTED_FAILURES = { "CWMOT5005W", "SRVE0315E", "SRVE0777E" };
+
+    @ClassRule
+    public static GenericContainer<?> container = new GenericContainer<>(new ImageFromDockerfile()
+                    .withDockerfileFromBuilder(builder -> builder.from(TestUtils.IMAGE_NAME)
+                                    .copy("/etc/otelcol-contrib/config.yaml", "/etc/otelcol-contrib/config.yaml"))
+                    .withFileFromFile("/etc/otelcol-contrib/config.yaml", new File(TestUtils.PATH_TO_AUTOFVT_TESTFILES + "config.yaml"), 0644))
+                    .withLogConsumer(new SimpleLogConsumer(JULLogServletTest.class, "opentelemetry-collector-contrib"))
+                    .withExposedPorts(4317, 4318);
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        TestUtils.trustAll();
+        WebArchive telemetryLogApp = ShrinkWrap
+                        .create(WebArchive.class, "MpTelemetryLogApp.war")
+                        .addPackage(
+                                    "io.openliberty.microprofile.telemetry.logging.internal.container.fat.MpTelemetryLogApp")
+                        .addAsManifestResource(new File("publish/resources/META-INF/microprofile-config.properties"),
+                                               "microprofile-config.properties");
+
+        ShrinkHelper.exportDropinAppToServer(server, telemetryLogApp,
+                                             DeployOptions.SERVER_ONLY);
+
+        server.addEnvVar("OTEL_EXPORTER_OTLP_ENDPOINT", "http://" + container.getHost() + ":" + container.getMappedPort(4318));
+
+        server.copyFileToLibertyServerRoot("agent-260/opentelemetry-javaagent.jar");
+        server.startServer();
+
+        //Read to run a smarter planet
+        server.waitForStringInLogUsingMark("CWWKF0011I");
+        server.setMarkToEndOfLog();
+    }
+
+    /**
+     * Ensure JUL message logs are not duplicated when the OpenTelemetry Java agent is active with mpTelemetry-2.0.
+     */
+    @Test
+    public void testMatchingJULMessageLogsWithContainerViaOpenTelemetryAgent() throws Exception {
+        assertTrue("The server was not started successfully.", server.isStarted());
+
+        TimeUnit.SECONDS.sleep(5);
+
+        final String logs = container.getLogs();
+
+        List<String> linesMessagesLog = server.findStringsInLogs("^(?!.*scopeInfo).*\\[.*$", server.getDefaultLogFile());
+        int bridgedLogsCount = logs.split("LogRecord #").length - 1;
+
+        assertEquals("Messages.log and Telemetry console logs don't match.", linesMessagesLog.size(), bridgedLogsCount);
```

Comment:
There is no console logs here, so better to change the message to "Telemetry bridged logs..."